frontend/src/Components/UseTitle/UseTitle.tsx

```diff
@@ -0,0 +1,7 @@
+import { useEffect } from 'react';
+
+export function UseTitle(title: string) {
```

Comment:
Gjerne bruk camelCase for funksjoner

```suggestion
export function useTitle(title: string) {
```